### DISCUSSION TITLE
Getting Problem In Local Setup In Windows 11

### DISCUSSION THREAD

**Discussion by sudhanshuku01** [KaTeX/KaTeX#3931] [General]
Please help me in resolving these issue getting during the local setup 😤

`PS C:\GitHub Contribution\KaTeX> yarn 
➤ YN0000: ┌ Resolution step
➤ YN0000: └ Completed in 0s 283ms
➤ YN0000: ┌ Fetch step
➤ YN0013: │ yargs-parser@npm:20.2.9 can't be found in the cache and will be fetched from the remote registry
➤ YN0013: │ yargs-parser@npm:21.1.1 can't be found in the cache and will be fetched from the remote registry
➤ YN0013: │ yargs@npm:16.2.0 can't be found in the cache and will be fetched from the remote registry
➤ YN0013: │ yargs@npm:17.5.1 can't be found in the cache and will be fetched from the remote registry
➤ YN0013: │ yocto-queue@npm:0.1.0 can't be found in the cache and will be fetched from the remote registry
➤ YN0000: └ Completed in 10s 876ms
➤ YN0000: ┌ Link step
➤ YN0000: │ ESM support for PnP uses the experimental loader API and is therefore experimental
➤ YN0007: │ husky@npm:4.3.8 must be built because it never has been before or the last one failed
➤ YN0009: │ husky@npm:4.3.8 couldn't be built successfully (exit code 1, logs can be found here: C:\Users\SUDHAN~1\AppData\Local\Temp\xfs-0884274d\build.log)
➤ YN0000: └ Completed in 33s 936ms
➤ YN0000: Failed with errors in 45s 301ms

PS C:\GitHub Contribution\KaTeX> yarn start
node:internal/process/esm_loader:40
      internalBinding('errors').triggerUncaughtException(
                                ^
Error [ERR_UNSUPPORTED_ESM_URL_SCHEME]: Only URLs with a scheme in: file, data, and node are supported by the default ESM loader. On Windows, absolute paths must be valid file:// URLs. Received protocol 'c:'
    at throwIfUnsupportedURLScheme (node:internal/modules/esm/load:236:11)
    at defaultLoad (node:internal/modules/esm/load:128:3)
    at nextLoad (node:internal/modules/esm/hooks:865:28)
    at load$1 (file:///C:/GitHub%20Contribution/KaTeX/.pnp.loader.mjs:175:12)
    at nextLoad (node:internal/modules/esm/hooks:865:28)
    at Hooks.load (node:internal/modules/esm/hooks:448:26)
    at handleMessage (node:internal/modules/esm/worker:196:24)
    at Immediate.checkForMessages (node:internal/modules/esm/worker:138:28)
    at process.processImmediate (node:internal/timers:478:21) {
  code: 'ERR_UNSUPPORTED_ESM_URL_SCHEME'
}

Node.js v20.10.0
PS C:\GitHub Contribution\KaTeX> `

**Comment by edemaine** [comment]
I haven't tried in a while, but perhaps the version of Yarn we're using doesn't support Node's new ESM loader system. Meanwhile, you might try using an older version of Node, e.g. via nvm.
